paras-marketplace-contract/src/lib.rs

```diff
@@ -1651,19 +1651,15 @@ impl Contract {
         let bidder_id = env::predecessor_account_id();
         let current_time = env::block_timestamp();
 
-        if market_data.started_at.is_some() {
-            assert!(
-                current_time >= market_data.started_at.unwrap(),
-                "Paras: Sale has not started yet"
-            );
-        }
+        assert!(
+            current_time >= market_data.started_at.unwrap(),
```

Comment:
it's possible to check the started_at and ended_at values only when is_auction true?